cl/lib/elasticsearch_utils.py

```diff
@@ -466,6 +496,33 @@ def build_es_base_query(search_query: Search, cd: CleanData) -> Search:
                 fields,
                 cd.get("q", ""),
             )
+        case SEARCH_TYPES.PEOPLE:
+            child_query_fields = {
+                "position": add_fields_boosting(
+                    cd, ["appointer", "supervisor", "predecessor"]
+                ),
+            }
+            parent_query_fields = add_fields_boosting(
+                cd,
+                [
+                    "name",
+                    "gender",
+                    "alias",
+                    "dob_city",
+                    "political_affiliation",
+                    "religion",
+                    "fjc_id",
+                    "aba_rating",
+                    "educations",
+                    "political_affiliation",
```

Comment:
It seems `political_affiliation` is duplicated here. Maybe it should be `political_affiliation_id`? However I don't see that `political_affiliation_id`  was in `person_text.txt`

Reply:
You're right!. `political_affiliation` is duplicated and we don't need `political_affiliation_id`. I will fix the list.